README.md

```diff
@@ -1,104 +1,128 @@
+[Pravega](http://pravega.io). We also provide code examples to connect analytics 
+engines such as [Flink](https://flink.apache.org/) and
+[Hadoop](http://hadoop.apache.org/) with Pravega as a storage substrate for data 
+streams. 
 
-## Getting Started
+For more information on Pravega, we recommend to read the [documentation and the
+developer guide](http://pravega.io).
 
-### Building Pravega
+# Repository Structure
 
-Optional: This step is required only if you want to use a different version
-of Pravega than is published to maven central.
+This repository is divided into sub-projects specifically addressed to demonstrate a
+single component (`standalone-examples` and `flink-examples`). 
+In each sub-project, we divided the examples into two categories: i) `features`, which are very simple
+examples aimed at demonstrating how a particular feature or API works; and ii)
+`scenarios`, which are more complex applications that show use-cases exploiting the 
+component at hand.
+
+## Pravega Examples
+| Example Name        |  Type  | Description  | Language |
+| ------------- |:-------------| :-----| :-----|
+| `gettingstarted` | feature | Simple example of how to read/write a Pravega `Stream`. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/gettingstarted)
+| `consolerw` | feature      | Application that allows users to work with `Stream`, `Transaction` and `StreamCut` APIs via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/consolerw)
+| `noop` | feature      | Example of how to add a simple callback executed upon a read event. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/noop)
+| `statesynchronizer` | feature | Application that allows users to work with `StateSynchronizer` API via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/statesynchronizer)
+| `streamcuts` | feature | Application examples demonstrating the use of `StreamCut`s via CLI. | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/streamcuts) 
+| `turbineheatsensor` | scenario | It emulates parallel sensors producing temperature values (writers) and parallel consumers performing real-time statistics (readers). | [Java](https://github.com/pravega/pravega-samples/tree/master/standalone-examples/src/main/java/io/pravega/example/turbineheatsensor)
+
+## Flink-connector Examples
+| Example Name        |  Type  | Description  | Language |
+| ------------- |:-------------| :-----| :-----|
+| `wordcount` | feature | Continuous count words on a Pravega `Stream` to demonstrate the usage of Flink connector. | [Java](https://github.com/pravega/pravega-samples/tree/master/flink-examples/src/main/java/io/pravega/examples/flink/wordcount)
```

Comment:
Continuous count words  -> `Counting the words continuously` from a Pravega Stream

Reply:
Corrected.